src/app/themes.tsx

```diff
@@ -29,6 +29,10 @@ const StyleManager: React.FC<Props> = ({ children }) => {
         return <>{styles}</>;
     });
 
+    if (typeof window !== "undefined") {
```

Comment:
can you rebase to (current) main to get rid of these sort of changes - which are cherrypicked when you needed them? :smile:

Reply:
Rebased as suggested!